scripts/add-global-changelog.js

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+
+*/
+
+import fs from 'fs';
+import { execSync } from 'child_process';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const repoUrl = 'https://github.com/adobe/spectrum-web-components';
+
+const pkg = JSON.parse(
+    fs.readFileSync(path.resolve(__dirname, '../package.json'), 'utf-8')
+);
+const newVersion = pkg.version;
+const newTag = `v${newVersion}`;
+const prevTag = execSync('git tag --sort=-creatordate')
+    .toString()
+    .split('\n')
+    .filter(Boolean)
+    .find((tag) => tag !== newTag);
```

Comment:
This needs more robust failure captures. I recommend if you're going to use exec for this, separate the command execution (exec is notoriously flaky in node scripts so you  need to account for it failing) from the string parsing. Check that exec returned a string and then run the split, etc.